framework/src/postprocessors/AreaPostprocessor.C

```diff
@@ -18,6 +18,9 @@ AreaPostprocessor::validParams()
 
   params.addClassDescription("Computes the \"area\" or dimension - 1 \"volume\" of a given "
                              "boundary or boundaries in your mesh.");
+  // Other user objects may depend on this value on INITIAL
+  // pre-initial conditions is as early as possible
+  params.set<bool>("force_preic") = true;
```

Comment:
I think this is ok, but I'd like to understand why it's necessary?

Reply:
because you run the flux BCs using the area pps on INITIAL because the RC UO is ran on INITIAL. 

We dont really need to run the flux BCs since the issue with VFRs is inside the domain, but detecting this on INITIAL and changing the list of residual object was harder

Reply:
Shouldn't pre-aux be sufficient? E.g. run at the "same time" as the RC UO? Because side user objects are run before general user objects

Reply:
yeah could be. I ll give it a try